cmd/translate/main.go

```diff
@@ -57,18 +74,18 @@ var rootCmd = &cobra.Command{
 		grpcL := m.Match(cmux.HTTP2())
 
 		go func() {
-			if err := server.Serve(httpL); err != nil {
+			if err = server.Serve(httpL); err != nil {
```

Comment:
You write to the same error instance from the main thread from two go routines, avoid it, it may cause panic. Use a local variable.

```suggestion
			if err := server.Serve(httpL); err != nil {
```